documentation/src/docs/streamcuts.md

```diff
@@ -0,0 +1,92 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Working with Pravega: StreamCuts
+
+This section describes about `StreamCut`s and how they can be used with streaming clients and batch clients.
```

Comment:
`This section describes StreamCuts` remove about.

Reply:
done.